src/pageObject.js

```diff
@@ -1,16 +1,39 @@
-'use strict';
+/* eslint-disable */
+"use strict";
 
 class Header {
-
-};
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
 class PageObject {
-
-};
+  constructor(url, header = new Header()) {
+    this.header = header;
+    this.url = url;
+  }
+  clickOnLogo() {
+    return "Click on the " + this.header.logo;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super();
```

Comment:
The `super()` call should include the `url` parameter to correctly initialize the `PageObject` base class. Consider using `super(url)`.